src/aspire/source/xform.py

```diff
@@ -182,6 +182,48 @@ def _forward(self, im, indices):
         return im.filter(self.filter)
 
 
+class Add(Xform):
+    """
+    A Xform that add the density of a stack of 2D images (in the form of an Image object)
+    by offsetting a number.
+    """
+    def __init__(self, mean):
```

Comment:
Use a less specific name instead of `mean`. Something simple like `im` or `x` might work. If you want something more specific, `addend` would do it.

Reply:
Changed to `addend`.